# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mtools/lseek64_proto_fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./llong.c.orig	Sat Aug 12 12:19:58 2000
+++ ./llong.c	Sat Aug 12 12:20:58 2000
@@ -45,7 +45,7 @@
 
 #if defined HAVE_LSEEK64
 # ifndef HAVE_LSEEK64_PROTOTYPE
-extern long long lseek64 (int fd, long long offset, int origin);
+// extern long long lseek64 (int fd, long long offset, int origin);
 # endif
 #endif
 
